AmericanMedicalAssociation/ama-style-guide-2
# [EWL-5045] Fix mobile load issue on Resources page

## Ticket(s)

**Jira Ticket**
- [EWL-5045: SG2 | Fix mobile load issue on Resources page](https://issues.ama-assn.org/browse/EWL-5045)


## Description
As a person viewing the Resources page on mobile, you should see that the "Resources" tab correctly displays by default.

This update adds a check on screen size to determine if the "Resources" tab should be active on page load. Script was added to twig file only to ensure it is run right at the moment the tab navigation element becomes part of the DOM. Placing code in any other place would cause a delay in which the tab is not properly highlighted until everything on the page is loaded.

There is a ticket logged to refactor the resources tab, [EWL-5018 SG2 | Code standards to-dos for Resource page](https://issues.ama-assn.org/browse/EWL-5018). I believe the refactoring could remove the need for this fix.

I have also included a css fix to overlapping text on the media tab on mobile.


## To Test
- Pull `bugfix/EWL-5045-mobile-load-issue-resources-page` branch
- Run `gulp serve` in ama-style-guide-2/styleguide to ensure a local version of style guide is running.
- Change the screen size to less than 600px and navigate to [Pages > Resource](http://localhost:3000/?p=pages-resource) and confirm resource tab is highlighted with appropriate content displayed.

## Visual Regressions

Updates to the mobile reference image have been included in this PR.


## Relevant Screenshots/GIFs
N/A


## Remaining Tasks
N/A


## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
